src/modules/task.js

```diff
@@ -0,0 +1,16 @@
+const todoTasks = [
+  {
+    completed: false,
+    index: 1,
+  },
+  {
+    completed: false,
+    index: 2,
+  },
+  {
+    completed: false,
+    index: 3,
+  },
+];
+
+export default todoTasks;
```

Comment:
- [ ] As requested by the previous reviewer, all hardcoded items from the tasks array are removed since you have implemented add task functionally and there is no need for you to have default todo tasks so please kindly remove these and make sure all functionality works as expected. :blush: